services/order.py

```diff
@@ -0,0 +1,63 @@
+
+from datetime import datetime
+from django.utils import timezone
+from django.db import transaction
+from django.core.exceptions import ValidationError
+from django.db.models import QuerySet
+
```

Comment:
According to the checklist, use `get_user_model()` instead of `User` for better practice when dealing with user models.